README.md

```diff
@@ -525,6 +527,7 @@ Global Flags:
       --cpu-profile-path string   Path to CPU profile output file
       --mem-profile-path string   Path to memory profile output file
   -v, --verbose                   verbose output
+      --version                   Print version
```

Comment:
I would include this change in another PR

Reply:
Removed.